org.eclipse.xtext.common.types.eclipse.tests/META-INF/MANIFEST.MF

```diff
@@ -1,65 +1,81 @@
-Manifest-Version: 1.0

```

Comment:
the old version has CR+LF, while the new has LF only.
We should clean this up in general. For this PR it is OK.

Reply:
or shall we do a cleanup first to see only the change here?

Reply:
i did nothing. this happens automatically

Reply:
=> i propose to simply have a manual look

Reply:
i have doubts the old one with the old manifest merger can be adopted to do the cleanup first